imblearn/metrics/pairwise.py

```diff
@@ -0,0 +1,162 @@
+"""Metrics to perform pairwise computation."""
+
+# Authors: Guillaume Lemaitre <g.lemaitre58@gmail.com>
+# License: MIT
+
+import numpy as np
+from scipy.spatial import distance_matrix
+
+
+class ValueDifferenceMetric:
```

Comment:
Should we derive from `BaseEstimator` since it is estimates?

Reply:
Yep this good a good idea.